.github/workflows/audit-security.yml

```
@@ -0,0 +1,48 @@
+# Copyright 2021 The Engula Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: Audit Security
+
+on:
+  push:
+    paths:
+      - '**/**/Cargo.toml'
+  pull_request:
+    paths:
+      - '**/**/Cargo.toml'
+  schedule:
+    - cron: '0 0 * * *'
```

```suggestion
    - cron: '25 4 * * *'
```

I assume many crontab job set as `'0 0 * * *'`. As we use public runner, I tend to use a random time.

Also maybe it's enough to run once a week. But I don't have a strong feeling on frequency.

---

once a week is enough

---

> As we use public runner, I tend to use a random time.

Good idea, from experience